sass/_themes.scss

```diff
@@ -1,73 +1,470 @@
 
 $themes: (
-  light: (
-    activeColor:            #4C5794,
-    defaultMain:            #fff,
-    defaultAccent:          #398dff,
-    defaultBg:              #f8f8f8,
-    defaultBgActive:        #f2f3f8,
-    defaultMenuBg:          #fff,
-    defaultSubmenuBg:       #f3f3f3,
-    defaultHuBg:            #ffffff,
-    defaultText:            #000,
-    defaultTextActive:      #4C5794,
-    defaultTextSubtle:      #999,
-    defaultTextMedium:      #666,
-    defaultTextStrong:      #000,
-    defaultBorder:          #e9e9ed,
-    defaultBorderSubtle:    #ddd,
-    defaultBorderMedium:    #ccc,
-    defaultBorderStrong:    #999,
-    chartGrid:              #f3f3f3,
-    chartText:              #cccccc,
-    chartBg:                #fff,
-    listBgActive:           #f2f3f8,
-    listBorder:             1px solid #e9e9ed,
-    controlText:            #575757,
-    controlTitleText:       #454545,
-    controlTitlebg:         #fff,
-    controlTextActive:      #4b5794,
-    controlActive:          #4b5794,
-    controlBgActive:        rgba(76, 87, 148, 0.2),
-    controlGrid:            #fafafb,
-    controlDialogHead:      #FFFFFF,
-    controlDialogBody:      #FAFAFA,
-    tooltipBg:              rgba(21,33,45,0.88),
-    tooltipTextColor:       #fff,
-  ),
-  dark: (
-    activeColor:            #e2e2e2,
-    defaultMain:            #1c2a35,
-    defaultAccent:          #151f28,
-    defaultBg:              #1C2A35,
-    defaultBgActive:        mix(#fff, #1c2a35, 8%),
-    defaultMenuBg:          #1c2a35,
-    defaultSubmenuBg:       #f3f3f3,
-    defaultHuBg:            #1c2a35,
-    defaultText:            #e7fafb,
-    defaultTextSubtle:      #e7fafb,
-    defaultTextMedium:      rgb(179, 174, 174),
-    defaultTextStrong:      #e7fafb,
-    defaultBorder:          #010103,
-    defaultBorderSubtle:    #ddd,
-    defaultBorderMedium:    #ccc,
-    defaultBorderStrong:    #999,
-    chartGrid:              mix(#fff, #1c2a35, 8%),
-    chartText:              mix(#fff, #1c2a35, 50%),
-    chartBg:                mix(#fff, #1c2a35, 10%),
-    listBgActive:           #e2e2e2,
-    controlText:            rgb(179, 174, 174),
-    controlTitleText:       #acacac,
-    controlTitlebg:         mix(#fff, #1c2a35, 7%),
-    controlActive:          #e2e2e2,
-    controlTextActive:      #e2e2e2,
-    controlBgActive:        rgba(76, 87, 148, 0.2),
-    controlGrid:            #fafafb,
-    controlDialogHead:      mix(#fff, #1c2a35, 3%),
-    controlDialogBody:      mix(#fff, #1c2a35, 7%),
-    tooltipBg:              #1c70cea1,
-    tooltipTextColor:       #fff,
-  ),
+    light: (
+        DefaultMain:            #ffffff,
```

Comment:
can you cut down the number of variables we need to tweak for the theme? It should not be specific to any component. This is waaaaaay too much. Discuss with Afiq on this. Something along the lines of a dozen colours will do.

Reply:
@brucebinary, of course, we can reduce the variables, but I act this way to separate all modules so that we have less change later.

Reply:
Ideally it shouldn't be like this.
However, we are still in the middle of developing the UI and I'm expecting lots of changes in colours and UI elements. 
So I think separating modules makes sense in this stage until things become stable and we can cut down the variables.

Reply:
@shayanbinary @ashkanx alright. We keep it as is then.